### PR TITLE
Enables unicorn project

### PR DESCRIPTION
**Pull request by catenacyber**
Following #1725 

@inferno-chromium I tested with `python infra/helper.py check_build unicorn` to make sure that all the targets are good to fuzz

### CODE REVIEW

File: projects/unicorn/build.sh [hunk 1]
```diff
@@ -27,6 +27,6 @@ do
 
     $CXX $CXXFLAGS fuzz_$target.o -o $OUT/fuzz_$target ../../libunicorn.a -lFuzzingEngine
 
-    #TODO corpuses
+    # TODO corpuses
     cp fuzz_emu.options $OUT/fuzz_$target.options
```
Comment (Dor1s) [inline projects/unicorn/build.sh]: If you really want to enforce `-max_len=4096` limit, you would have to do it in the code. Please see https://github.com/google/oss-fuzz/blob/22f3a1481cbc8252fb340369cdb3d50f185017fa/docs/new_project_guide.md#custom-libfuzzer-options-for-clusterfuzz

**Comment by catenacyber** [comment]
@inferno-chromium I thought I had to reenable it so that it can get built once more. Is this the other way around ?

Looking at the build log, I proposed one fix
https://github.com/unicorn-engine/unicorn/pull/1011

How come I did not get this error locally ?

**Comment by inferno-chromium** [comment]
This is not a build failure, but a build check failure that ensures build is good.
https://github.com/google/oss-fuzz/blob/master/docs/reproducing.md#reproducing-build-failures

**Comment by catenacyber** [comment]
Thanks @infernochromium i had only checked the build with address sanitizer

**Pull request merged**